src/autopas/selectors/FeatureVector.h

```diff
@@ -164,6 +164,32 @@ class FeatureVector : public Configuration {
 
     return result;
   }
+
+  /**
+   * Create n latin-hypercube-samples from given featureSpace only considering continuous values and append a value
+   * representing the current iteration.
+   * @param n number of samples
+   * @param rng
+   * @param cellSizeFactors
+   * @param iteration current iteration which may be scaled by some factor
+   * @return vector of sample featureVectors
+   */
+  static std::vector<Eigen::VectorXd> lhsSampleFeatureContinuousWithIteration(size_t n, Random &rng,
+                                                                              const NumberSet<double> &cellSizeFactors,
+                                                                              double iteration) {
+    // create n samples from each set
+    auto csf = cellSizeFactors.uniformSample(n, rng);
+
+    std::vector<Eigen::VectorXd> result;
+    result.reserve(n);
+    for (size_t i = 0; i < n; ++i) {
+      Eigen::VectorXd vec(featureSpaceContinuousDims + 1);
+      vec << csf[i], iteration;
```

Comment:
Question:
If I understand this `<<` operator correctly, you always put exactly two values in `vec`. Why not use an `std::array<double, 2>` or a `std::pair`? Do you need the `VectorXd` type for the Algorithm later on?